srcpkgs/nextcloud-client/template

```diff
@@ -39,6 +39,14 @@ if [ "$CROSS_BUILD" ]; then
 	# provides desktoptojson
 	hostmakedepends+=" $(vopt_if dolphin 'kcoreaddons')"
 fi
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	#provides argp library
+	makedepends+=" argp-standalone"
+fi
+if [ "$XBPS_TARGET_MACHINE" != "armv7l" ]; then
+	#these packages are broken on that architecture
+	hostmakedepends+=" qt5-devel qt5-webengine-devel qt5-webview-devel"
+fi
```

Comment:
Did you see this build break?

Reply:
before when building it said that it was unable to build the package as it was broken for some architecture, but it appears to be working now and i was unable to reproduce it so i removed those lines